### PR TITLE
Re-add `AutoGraphicsPower`

### LINKED ISSUE #280: Error on `system76-power graphics power auto`
**Distribution (run `cat /etc/os-release`):** Pop!_OS 21.04 and 20.04 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):** efd0754 (current `master` branch)

**Issue/Bug Description:** When a system is in Compute graphics mode, the discrete graphics can be turned off and on using `system76-power graphics power off` and `system76-power graphics power on`. The help text for `system76-power graphics power` shows that a third setting, `auto`, is available, but when attempting to switch to that mode, I get the following output:

```
setting discrete graphics to turn off when not in use
daemon returned an error message: "Unknown method AutoGraphicsPower"
```

This is different from the message shown when a nonexistent mode is specified.

**Steps to reproduce (if you know):** Run `system76-power graphics power auto` on any system (switchable or non-switchable.)

**Expected behavior:** Either the command should complete successfully, or it should be removed/hidden from the help text.

**Other Notes:** N/A

### PR DESCRIPTION
**Pull request by DianaNites**
Fixes #280

### CODE REVIEW

File: src/daemon.rs [hunk 1]
```diff
@@ -259,6 +259,7 @@ pub async fn daemon() -> Result<(), String> {
         sync_get_method(b, "GetSwitchable", "switchable", PowerDaemon::get_switchable);
         sync_get_method(b, "GetGraphicsPower", "power", PowerDaemon::get_graphics_power);
         sync_set_method(b, "SetGraphicsPower", "power", PowerDaemon::set_graphics_power);
+        sync_get_method(b, "AutoGraphicsPower", "thresholds", PowerDaemon::auto_graphics_power);
```
Comment (jackpot51) [inline src/daemon.rs:262]: I think `thresholds` should be `power`

Comment (DianaNites) [reply src/daemon.rs]: Fixed

Comment (DianaNites) [reply src/daemon.rs]: Actually, should it? In the original code it had nothing

https://github.com/pop-os/system76-power/pull/195/files#diff-bf435d9dfa9a63896065cbbd3082d65b24a937afbab2638ae820385108446a59L275

**Review by jackpot51 — APPROVED** [review]
Looks good now, thanks!

**Pull request closed without merging**